ohmec_data_na.js

```diff
@@ -4,6 +4,146 @@ dataNA = {
   "_comment3":"SPDX-License-Identifier: Apache-2.0",
   "type":"FeatureCollection",
   "features":[
+    { "type":"Feature",
+      "id":"OH700",
+      "properties":{
+        "entity1type":"nation",
+        "entity1name":"Native American",
+        "entity2type":"people",
+        "entity2name":"Ohlone",
+        "labelRotate":45,
+        "labelY":0,
```

Comment:
I would leave out anything that is default just to keep the file size at a minimum. Some day this filesize will be the death of us...